src/jit/lowerxarch.cpp

```diff
@@ -2654,7 +2654,7 @@ bool Lowering::IsContainableHWIntrinsicOp(GenTreeHWIntrinsic* containingNode, Ge
                 case NI_Base_Vector256_CreateScalarUnsafe:
                 {
                     // These intrinsics require the contained operand to be the same size as the widened
-                    // baseType (genActualType(baseType) of the containing node. This is because codegen 
+                    // baseType (genActualType(baseType) of the containing node. This is because codegen
```

Comment:
nit: looks like there's a missing parenthesis after `(genActualType(baseType)`

That said, I'm not quite sure what this comment is trying to say.

Reply:
The mention of "actual type" is misleading. This has nothing to do with the typical JIT type system behavior, it's just that the generated instruction (MOVD) doesn't support anything else.

Reply:
Actually, I think it *is* related to the usual JIT type system behavior - in that this is an intrinsic for which all cases support only the "JIT actual type" (in the floating point as well as integer case). This is pretty common for codegen, and I think it's why `genActualType` has the codegen prefix. However, I can see how it's not totally clear so I'll attempt to reword the comment.

Reply:
> in that this is an intrinsic for which all cases support only the "JIT actual type"

Yes, but that's pretty much a coincidence. `MOVD` could have supported only byte and dword operands. Or only pointer sized operands. And so on. Oh well, perhaps it doesn't matter that much since this applies to the "base type" of a SIMD node and not to the `gtType` of a random node. I'm just concerned that people seem to have been confused about JIT's "actual type" in the past and random useless comments won't help.

> However, I can see how it's not totally clear so I'll attempt to reword the comment.

I'd delete most of it. I find it strange to have such a large comment that explains what's pretty much obvious - yeah, you can't contain an operand if it doesn't have the proper size.